cypher/cypher-docs/src/test/scala/org/neo4j/cypher/docgen/FulltextIndexTest.scala

```diff
@@ -175,6 +178,25 @@ class FulltextIndexTest extends DocumentingTest {
       p("A complete description of the Lucene query syntax can be found in the link:https://lucene.apache.org/core/8_2_0/queryparser/org/apache/lucene/queryparser/classic/package-summary.html#package.description[Lucene documentation].")
     }
 
+    section( "Handling of Text Array properties", id="administration-indexes-fulltext-search-query") {
```

Comment:
```suggestion
section( "Handling of Text Array properties", id="administration-indexes-fulltext-search-text-array-properties") {
```

Reply:
luckily I have another change in this file and fixed this section id too https://github.com/neo4j/neo4j-documentation/pull/1569